src/util/util.js

```diff
@@ -34,6 +34,10 @@ export function isWindowsDriver() {
     return (store.getConfig().driver_interface.interface === "TUSB");
 }
 
+export function isWindowsWithDevice() {
```

Comment:
I'd replace this with something like:

```javascript
export function hasDriverVersion() {
  return store.getConfig().driver_interface.version !== undefined
}
```

And use that as a check instead. I dislike having windows specific checks labelled as windows specific in the naming convention for fields which are generic. Even if this is a specific windows fix, it's still a generic check.